cves/CVE-2015-8662.yml

```diff
@@ -64,7 +64,12 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A function that decodes JPEG2000 compression does not check the level of 
+  compression on the JPEG2000 file being decompressed. This can lead to a situation
```

Comment:
Not quite sure what you mean by "the level of compression on the JPEG2000 file", maybe add some clarification here.